core/traits/long_neck.py

```diff
@@ -0,0 +1,18 @@
+from core.constants import Signal, TraitCategory
+from core.trait_card import TraitCard
+from pydispatch import dispatcher
+
+
+class LongNeck(TraitCard):
+    def __init__(self):
+        super().__init__(type(self).__name__, tuple([TraitCategory.EATING]))
```

Comment:
For clarity, I suggest: a) assigning these two arguments to two variables above the super() call; b) passing these variables to super(). What do you think?

Reply:
Makes sense, done.